src/game/gameflow.c

```diff
@@ -508,6 +508,12 @@ static bool GameFlow_LoadLevelSequence(
         } else if (!strcmp(type_str, "remove_scions")) {
             seq->type = GFS_REMOVE_SCIONS;
 
+        } else if (!strcmp(type_str, "remove_ammo")) {
+            seq->type = GFS_REMOVE_AMMO;
+
+        } else if (!strcmp(type_str, "remove_medpacks")) {
+            seq->type = GFS_REMOVE_MEDPACKS;
```

Comment:
I'm going to be annoying and ask if we can change all the `medpack`s to `medipack`s to match the in game name / rest of the code.

Reply:
> Code looks good to me. Did you test a gameflow with different combinations of removals? Just want to make sure there's no strange bugs like the Scion removal equipping weapons. From just looking at the code, I don't see any potential issues but you never know ha.

Yes, I tested with what the randomizer will generate and it all looks good. See example below - so my testing method was start the level, cheat to get everything, move to next and check it matches the gameflow.
[Tomb1Main_gameflow.json5.txt](https://github.com/rr-/Tomb1Main/files/9410349/Tomb1Main_gameflow.json5.txt)
 
> I'm going to be annoying and ask if we can change all the `medpack`s to `medipack`s to match the in game name / rest of the code. In game technically uses `Medi Pack` but `medipack` is fine for the code IMO.

That's not a problem and it makes sense. I will update it.